lms/static/sass/base/_layouts.scss

```diff
@@ -3,7 +3,8 @@
 
 // overriding existing styles on the body element
 // .view-incourse scopes these rules to be specific to student being in a course
-body.view-incourse {
+body.view-incourse,
+body.is-in-course {
```

Comment:
I noticed that both teams and student notes uses the "is-in-course" body class instead of "view-incourse" - they should be made consistent, which I can do now if you think it makes sense to do it here.